include/zephyr/drivers/spi.h

```diff
@@ -102,8 +86,6 @@ extern "C" {
  * @name Specific SPI devices control bits
  * @{
  */
-/* Requests - if possible - to keep CS asserted after the transaction */
-#define SPI_HOLD_ON_CS		BIT(12)
```

Comment:
Maybe you should move the removals above into the next patch:  `include: zephyr: dt-bindings: spi: move CPOL, CPHA, HOLD_CS declarations`. Else all these definitions are missing between the two patches and this could break git-bisect.

Reply:
Thanks for your comment. I merged the 2 commits.